### ISSUE TITLE
Installation about `og_marl`

### ISSUE THREAD

**Issue by fyqqyf** [zbzhu99/madiff#2]
When executing `pip install -r ./third_party/og_marl/requirements.txt`, an error occurs indicating 'No such file or directory: './third_party/og_marl/requirements.txt'.

**Comment by zbzhu99** [comment]
Try the updated command (https://github.com/zbzhu99/madiff/commit/ddef3ab57a6e149f97b9dcb4136042dbf290516c):

```bash
pip install -r ./third_party/og-marl/install_environments/requirements/smacv1.txt
```

**Comment by zbzhu99** [comment]
Sorry for the mismatch between `og-marl` package and transformation script. Please pull the latest commit and reinstall the local `og-marl` package, which has the version of 0.0.1. This should work fine with current script and data.

**Comment by zbzhu99** [comment]
Oops, I only tested the transformation of smac datasets. The problem on mamujoco datasets can be fixed with a minor change: https://github.com/zbzhu99/madiff/commit/bd0c5b7ab834b7b8c1912653ae06f9c7dc989ccf

**Comment by zbzhu99** [comment]
It seems the developers of og-marl updated the data in their download links. In my local folder downloaded before, the files look like this:
![image](https://github.com/zbzhu99/madiff/assets/32090492/731e709a-1953-4efd-bb74-6c7084757060)
I am not sure whether they just rearrange the folders or provide a different dataset. I will take a look and upload my local copy if necessary.

**Comment by zbzhu99** [comment]
@fyqqyf , I have checked og-marl's new dataset and found they just renamed and rearranged the log folders from different tasks to a unified structure. 

Actually, I heard from the maintainer of og-marl dataset that they are also implementing the transformation (concatenate segments to complete trajectories) I have done here. So, the change of file structure may be to make it easier to process all datasets with the same script.

I have modified the script `scripts/transform_og_marl_dataset.py` and it should work fine with all new datasets now. Still, I will provide links for downloading transformed datasets by the end of this week.

*Happy Women's Day!*
